go/vt/mysqlctl/backup.go

```diff
@@ -97,7 +97,7 @@ var (
 )
 
 func init() {
-	for _, cmd := range []string{"mysqlctl", "mysqlctld", "vtcombo", "vttablet", "vttestserver", "vtbackup", "vtctld", "vtctldclient", "vtexplain"} {
+	for _, cmd := range []string{"vtcombo", "vttablet", "vttestserver", "vtbackup", "vtctld", "vtctldclient"} {
```

Comment:
vtctldclient can be removed from this list.

Reply:
done